lib/src/json_schema/schema_type.dart

```diff
@@ -39,21 +39,21 @@
 class SchemaType implements Comparable<SchemaType> {
   const SchemaType._(this.value);
 
-  static const SchemaType ARRAY = const SchemaType._(0);
+  static const SchemaType array = const SchemaType._(0);
```

Comment:
Just curious, why change everything to lowercase?

Reply:
https://www.dartlang.org/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names